tests/e2e_test.go

```diff
@@ -373,3 +373,88 @@ func TestPublicDBConnections(t *testing.T) {
 	}
 
 }
+
+func TestUpdateWorkloadOnDelete(t *testing.T) {
+
+	const (
+		pwlName  = "newss"
+		appLabel = "busybox"
+		name     = "app"
+		allOrAny = "all"
+	)
+	// Use a deployment workload
+	wl := &workload.DeploymentWorkload{Deployment: testhelpers.BuildDeployment(types.NamespacedName{}, "busybox")}
+	o := wl.Object()
+	kind := o.GetObjectKind().GroupVersionKind().Kind
+
+	// Set up the e2e test namespace
+	skipCleanup := loadValue("SKIP_CLEANUP", "", "false") == "true"
```

Comment:
Why doesn't `loadValue` just return a bool?

Reply:
This is used in other places where it needs to return the contents of the env var.

Reply:
Could generics make this less stringy?